src/main/java/no/difi/statistics/elasticsearch/commands/CategoriesQuery.java

```diff
@@ -0,0 +1,108 @@
+package no.difi.statistics.elasticsearch.commands;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.difi.statistics.model.OwnerCategories;
+import org.apache.http.util.EntityUtils;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.RestClient;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CategoriesQuery {
+
+    private RestClient elasticSearchClient;
+
+    private CategoriesQuery() {
+    }
+
+    public Set<OwnerCategories> execute() throws IOException {
+        // Put index-name and categories in Map, and copy to a Set before returning.
+        Map<String, OwnerCategories> ownerCategoriesMap = new HashMap<>();
+
+        Request request = new Request("GET", "/*@*@*/_mapping");
+        String mappings = EntityUtils.toString(elasticSearchClient.performRequest(request).getEntity());
+
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
ObjectMapperen kan være en instance-variable istedet for en local-variable så slipper den å instansieres hver gang metoden kalles. Altså flyttes ut av metoden og instansieres sammen med klassen.

Reply:
Da er den gjort til en klasse-variabel.